src/test/java/automation/tests/ManagingGroupTest.java

```diff
@@ -0,0 +1,75 @@
+package automation.tests;
+
+import automation.utils.loaders.EnvironmentConfigLoader;
+import automation.utils.loaders.GroupPropLoader;
+import automation.utils.loaders.UserConfigLoader;
+import automation.pages.BrowsePage;
+import automation.pages.LoginPage;
+import automation.pages.NewGroupPage;
+import org.junit.jupiter.api.*;
+import org.openqa.selenium.WebDriver;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertTrue;
+
+public class ManagingGroupTest {
+
+    private static LoginPage loginPage;
+    private static WebDriver driver;
+    private static UserConfigLoader userConfLoader = new UserConfigLoader("user");
+    private static EnvironmentConfigLoader envConfLoader = new EnvironmentConfigLoader("environment");
+    private NewGroupPage newGroupPage;
+    private BrowsePage browsePage;
+    private GroupPropLoader groupPropLoader = new GroupPropLoader("group");
+
+    //TC01 - New group can be added
+    @Test
+    void checkIfNewGroupCanBeCreated() {
```

Comment:
Why is your test named "TC01 - New group can be added", but your test method is named "checkIfNewGroupCanBeCreated"? They must match as much as possible (keeping in mind limitations on java test names, of course).

Reply:
Done